src/styles/blocks/card.scss

```
@@ -0,0 +1,115 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 0 16px;
+  max-width: $card-windth;
```

Also fix typo in variable's name
```suggestion
  width: $card-width;
```